apache/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG - apache
 
+1.1.0 / Unreleased
+==================
+### Changes
+
+* [UPDATE] Update auto_conf template to support agent 6 and 5.20+. See [#860][]
```

Comment:
@xvello didn't you forgot the actual link on all the changelogs? 🤔 

`[#860][]` in plaintext in rendered markdown

Reply:
That's the pimpmychangelog default format, the actual link will be added by the release script